src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -313,7 +313,14 @@ private void displaySlimefunItem(ChestMenu menu, ItemGroup itemGroup, Player p,
                         if (sfitem instanceof MultiBlockMachine) {
                             Slimefun.getLocalization().sendMessage(pl, "guide.cheat.no-multiblocks");
                         } else {
-                            pl.getInventory().addItem(sfitem.getItem().clone());
+                            if (action.isShiftClicked()) {
+                                ItemStack sfItem = sfitem.getItem().clone();
+                                sfItem.setAmount(sfItem.getMaxStackSize());
+
+                                pl.getInventory().addItem(sfItem);
+                            } else {
+                                pl.getInventory().addItem(sfitem.getItem().clone());
+                            }
```

Comment:
```suggestion
                            ItemStack clonedItem = sfitem.getItem().clone();
                            if (action.isShiftClicked()) {
                                clonedItem.setAmount(clonedItem.getMaxStackSize());

                                pl.getInventory().addItem(clonedItem);
                            } else {
                                pl.getInventory().addItem(clonedItem);
                            }
```
You can extract this duplicate part and I'd also give this a better name. `sfitem` and `sfIteme` are far too similiar